core/iwasm/aot/aot_loader.c

```diff
@@ -1432,6 +1587,8 @@ load_types(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
             func_type->base_type.type_flag = type_flag;
             func_type->base_type.is_sub_final = is_sub_final;
             func_type->base_type.parent_type_idx = parent_type_idx;
+            func_type->base_type.rec_count = rec_count;
+            func_type->base_type.rec_idx = rec_idx;
```

Comment:
Seems that we can add a function to set these 5 fields, and call it for func_type/struct_type/array_type.

Reply:
Done